crates/driver/src/infra/solver/dto/notification.rs

```diff
@@ -108,6 +118,10 @@ pub enum Kind {
         succeeded_once: bool,
     },
     InvalidClearingPrices,
+    MissingPrice {
```

Comment:
```suggestion
    #[serde(rename_all = "camelCase")]
    MissingPrice {
```

Reply:
This is redundant. The enum `Kind` already derives it.

Reply:
The `derive` on the enum kind only causes the enum variant **names** to be camelcased but not the **fields** of the structs inside the variant.

Reply:
ah, I misunderstood, in that case yes.